lsp/lsp.cpp

```diff
@@ -34,11 +34,8 @@
 #include <sstream>
 #include <fstream>
 
-// Begin log
-std::ofstream logfile;
-
-#include "json5.h"
-#include "execpath.h"
+#include <json5.h>
+#include <execpath.h>
```

Comment:
In the wake codebase, I use `"foo.h"` to denote a header file in the wake git repository and `<bar.h>` to denote system libraries. Therefore, please do not make this change.

Reply:
Done.